benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -157,6 +157,12 @@ protected BenchmarkServiceFutureStub build(io.grpc.Channel channel,
     }
   }
 
+  public static abstract class AbstractBenchmarkService implements BenchmarkService, io.grpc.stub.BindableService {
```

Comment:
One use case, on which I'm blocked because of lack functionality like this, is in our framework where we define exposed grpc services via annotations. Simplified example:

```java
@App("route-guide")
@ExposedGrpcService(RouteGuideService.class)
class RouteGuideApp {
  ...
}

class RouteGuideService extends AbstractRouteGuide {
 ...
}
```

With `BindableService` interface, we can do strong typing for `value` in `@ExposedGrpcService` - `Class<? extends BindableService>` - and avoid using reflection or other weird and tangled logic to build grpc server,  it'll be as simple as:

```java
ServerBuilder.forPort(...).addService(exposedGrpcServiceAnnotation.value().bindService())
```

I can imagine people having a number of similar use cases in their frameworks/apps where richer type for service implementation will be beneficial. It will also be helpful when using DI frameworks etc.